clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -470,6 +470,55 @@ check_sane_control_flow()
             return false;
         }
     }
+    // Correct test: back-to-back signals after an RSEQ abort.
+    {
+        std::vector<memref_t> memrefs = {
+            gen_instr(TID, /*pc=*/101, /*size=*/1),
+            // The RSEQ_ABORT marker is always follwed by a KERNEL_EVENT marker.
+            gen_marker(TID, TRACE_MARKER_TYPE_RSEQ_ABORT, 102),
+            gen_marker(TID, TRACE_MARKER_TYPE_KERNEL_EVENT, 102),
+            // We get a signal after the RSEQ abort.
```

Comment:
This is actually the signal that *caused* the rseq abort (otherwise the marker values would have to match) so I would reword this comment.

(102 is the rseq interruption point, which != handler only for signal interruption; 301 is rseq handler which is the signal resumption point)

Reply:
Done.